datafusion/src/physical_plan/planner.rs

```diff
@@ -791,6 +791,13 @@ impl DefaultPhysicalPlanner {
                         "Unsupported logical plan: CreateExternalTable".to_string(),
                     ))
                 }
+                | LogicalPlan::CreateMemoryTable {..} => {
+                    // Create a dummy exec.
```

Comment:
I think it might make more sense here to throw an error about "unsupported logical plan" -- if we return an `EmptyExec` that will result in a plan that does nothing (aka doesn't actually create any tables / table providers) but doesn't error which might be surprising

Reply:
I tried that before, and it failed I think somehow at the planning stage?